documentation-website/src/components/default-meta.tsx

```diff
@@ -28,7 +28,7 @@ const DefaultMeta = ({
         <Head.Meta
           content={description}
           name='description'/>
-        <Head.Title>{title} | @idrinth/api-bench</Head.Title>
+        <Head.Title>{title} | @idrinth-api-bench/api-bench</Head.Title>
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted